File: b_rank_skillcheck_archive/plural_form_04.cs
// 実行時間 20ms
using static System.Console;

internal class Program
{
    static void Main()
    {
        int n = int.Parse( ReadLine() );

        for( int _ = 0; _ < n; _++ ) {
            string a = ReadLine();

            if( a.EndsWith( "y" ) ) {
                if( !a.EndsWith( "ay" ) && !a.EndsWith( "iy" ) && !a.EndsWith( "uy" ) && !a.EndsWith( "ey" ) && !a.EndsWith( "oy" ) ) {
                    WriteLine( $"{a.Substring( 0, a.Length - 1 )}ies" );
                } else {
                    WriteLine( $"{a}s" );
                }
            } else {
                WriteLine( $"{a}s" );
            }
        }
        return;
    }
}

File: b_rank_skillcheck_archive/plural_form_04.py
# 実行時間 10ms
n = int(input())

for _ in [0] * n:
    a = input()

    if a.endswith("y"):
        if a.endswith(("ay", "iy", "uy", "ey", "oy")) != True:
            print(f"{a[:-1]}ies")
        else:
            print(f"{a}s")
    else:
        print(f"{a}s")
